lib/cdo/workshop_constants.rb

```diff
@@ -0,0 +1,35 @@
+module WorkshopConstants
```

Comment:
Before doing that I would check that the ops dashboard ui (which is in
pegasus) doesn't use them.

On Tue, Nov 17, 2015 at 8:21 PM, Andrew Oberhardt <notifications@github.com>
wrote:

> In lib/cdo/workshop_constants.rb
> <https://github.com/code-dot-org/code-dot-org/pull/5472#discussion_r45158593>
> :
>
> > @@ -0,0 +1,35 @@
> > +module WorkshopConstants
>
> No idea. These were in ActivityConstants before, which was not the right
> place. It follows that shared is not the right place either. I'll move them
> to dashboard.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/code-dot-org/code-dot-org/pull/5472/files#r45158593>.
>